master/pkg/schemas/expconf/experiment_config.go

```diff
@@ -241,12 +251,22 @@ type BindMountV0 struct {
 type DevicesConfigV0 []DeviceV0
 
 // Merge implements the Mergable interface.
-func (b DevicesConfigV0) Merge(other interface{}) interface{} {
+func (d DevicesConfigV0) Merge(other interface{}) interface{} {
 	tOther := other.(DevicesConfigV0)
 	// Merge by appending.
 	out := DevicesConfigV0{}
-	out = append(out, b...)
-	out = append(out, tOther...)
+	out = append(out, d...)
+
+	// Prevent duplicate container paths as a result of the merge.
+	paths := map[string]bool{}
```

Comment:
could drop a comment that `DevicesConfigV0.Merge` prefers the receiver, not the param's devices/bind mounts, if there are multiple devices/bind mounts for a single container path.

Reply:
oops, I blindly landed this without seeing your comment, sorry.